### ISSUE TITLE
Regarding Model Training

### ISSUE THREAD

**Issue by jaytimbadia** [MaharshSuryawala/Food-Image-Recognition#3]
Very good work! Really appreciated.

I just started training this model and its seems that, its training even inception Net from scratch, right? 
As I can see there is no trainable = False for base model.

Also, while I am training, the accuracy at start is 1%, which means, is it training from scratch?
If yes, than, what does weights='imagenet' means? its pre-trained weights. I am confused here.

```
inception = InceptionV3(weights='imagenet', include_top=False)
layer = inception.output
layer = GlobalAveragePooling2D()(layer)
layer = Dense(128,activation='relu')(layer)
layer = Dropout(0.2)(layer)

```

Any help is much appreciated.

![image](https://user-images.githubusercontent.com/39621190/154831627-c552069a-0c45-4199-81c0-d33f03662b11.png)

**Comment by MaharshSuryawala** [comment]
Hi,

Thanks for reaching out.

The model is trainable by default. The weights='imagenet' just means that it initializes the weights to the pre-trained weights of imagenet, if that value is set to "None" it will initialize the weights randomly and the model will be trained from scratch accordingly. For more information about it please refer to [this](https://keras.io/api/applications/inceptionv3/) link. Let me know if you have any more questions.

Sincerely,
Maharsh
